cypress/e2e/dashboard_spec.cy.js

```diff
@@ -5,33 +5,32 @@ describe('Dashboard flows', () => {
     cy.visit('http://localhost:3000/')
   })
   
-  it('Should display the app title on page load', () => {
-    cy.get('h1')
-    .contains('Rancid Tomatillos')
+  it('User should see the app title on page load', () => {
+    cy.get('h1').contains('Rancid Tomatillos')
```

Comment:
We should probably add a data-cy attribute to this element for the sake of consistency.  I can take care of that.